code-experiments/build/matlab/exampleexperiment.m

```diff
@@ -31,7 +31,7 @@
 % bbob-biobj-ext    92 unconstrained noiseless bi-objective functions
 % bbob-largescale   24 unconstrained noiseless single-objective functions in large dimensions
 %
-suite_name = 'bbob';
+suite_name = 'bbob-constrained';
```

Comment:
Same here: I think we should keep the `bbob` suite the default one and mention the `bbob-constrained` suite in the comment above instead.